troposphere/static/js/components/identities/detail/resources/IdentityResourcesWrapper.jsx

```diff
@@ -1,7 +1,6 @@
 import React from "react";
 import Backbone from "backbone";
-//FIXME: these imports shouldnt _require_ using ../../.. 
-import SubMenu from "../../../projects/detail/resources/SubMenu";
+
```

Comment:
I believe the `<SubMenu />` has one usage within the Project Details, but I will confirm.